tasks/main.yml

```diff
@@ -161,13 +121,6 @@
     - deploydrupal_npm_theme_build
     - deploydrupal_npm_theme_path is defined
 
-- name: clear caches.
```

Comment:
Don't we still need this after theme files are rebuilt? If the cache was set between the commands above and before theme building finished, we could have the wrong things in the cache, right?

Reply:
I suppose that is true. Another option: What if we rebuilt the theme first?

Reply:
I'm all for that. @agarzola any concerns?

Reply:
I don’t see a problem with building the theme first. Go for it!

Reply:
This seems like a big change for some reason (makes me nervous). Let's see how it goes!